src/tools-evaluation.md

```diff
@@ -38,6 +38,21 @@ terms of arguments and flags and possibly additional utilities they include.
 | [u-root](https://u-root.org)                             | Go       | BSD 3-Clause | ByteDance, Google et al |
 | [uutils/coreutils](http://uutils.github.io/)             | Rust     | MIT          | not for LinuxBoot       |
 
+### Boot menus
+
+There are multiple approaches to defining boot menus, such as GRUB's
+configuration files, EXTLINUX, and [Boot Loader Specification (BLS)](https://uapi-group.org/specifications/specs/boot_loader_specification/).
```

Comment:
Maybe reword since those things are not boot menus, but rather descriptions of bootable operating systems. In BLS it looks like they refer to them as "boot loader menu entries".  It looks like in EXTLINUX/syslinux just refer to them as their configuration files.